DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/recoverable_sysctl.py

```diff
@@ -0,0 +1,43 @@
+from dent_os_testbed.lib.os.sysctl import Sysctl
+
+
+class RecoverableSysctl(Sysctl):
+
+    saved_values = dict()
+
+    async def set(*argv, **kwarg):
+        devices = kwarg['input_data']
+        kwarg['parse_output'] = True
```

Comment:
What do you think about calling `Sysctl.get()` without the `options` argument? So that we always have the variable and the value.

Reply:
Please see lines 12-17, I think that's what you mean

Reply:
yes, my bad